keep-ui/app/workflows/builder/page.client.tsx

```diff
@@ -62,78 +64,85 @@ export default function PageClient({
+              Beta
+            </Badge>
           </Title>
           <Subtitle>Workflow building kit</Subtitle>
         </div>
-        {workflow ? (
-          <div>
-            <Button
-              color="orange"
-              size="md"
-              icon={ArrowUpOnSquareIcon}
-              disabled={!generateEnabled}
-              onClick={() => setTriggerSave(triggerSave + 1)}
-            >
-              Deploy
-            </Button>
-          </div>
-        ) : (
-          <div>
-            <Button
-              color="orange"
-              size="md"
-              className="mr-2"
-              onClick={newAlert}
-              icon={PlusIcon}
-              variant="secondary"
-              disabled={!buttonsEnabled}
-            >
-              New
-            </Button>
-            <Button
-              color="orange"
-              size="md"
-              className="mr-2"
-              onClick={loadAlert}
-              variant="secondary"
-              icon={ArrowDownOnSquareIcon}
-              disabled={!buttonsEnabled}
-            >
-              Load
-            </Button>
-            <input
-              type="file"
-              id="alertFile"
-              style={{ display: "none" }}
-              onChange={handleFileChange}
-            />
-            <Button
-              color="orange"
-              size="md"
-              className="mr-2"
-              icon={ArrowUpOnSquareIcon}
-              disabled={!generateEnabled}
-              onClick={() => setTriggerSave(triggerSave + 1)}
-            >
-              Deploy
-            </Button>
+        <div className="flex gap-2">
+          {workflow && (
+            <>
+              <Button
+                color="orange"
+                size="md"
+                onClick={newAlert}
+                icon={PlusIcon}
+                variant="secondary"
+                disabled={!buttonsEnabled}
+              >
+                New
+              </Button>
+              <Button
+                color="orange"
+                size="md"
+                onClick={loadAlert}
+                variant="secondary"
+                icon={ArrowDownOnSquareIcon}
+                disabled={!buttonsEnabled}
+              >
+                Load
+              </Button>
+              <input
+                type="file"
+                id="alertFile"
+                style={{ display: "none" }}
+                onChange={handleFileChange}
+              />
+            </>
+          )}
+          <Button
+            color="orange"
+            size="md"
+            icon={PlayIcon}
+            disabled={!generateEnabled}
+            onClick={() => setTriggerRun(incrementState)}
+          >
+            Run
+          </Button>
+          <Button
+            color="orange"
+            size="md"
+            icon={ArrowUpOnSquareIcon}
+            disabled={!generateEnabled}
+            onClick={() => setTriggerSave(incrementState)}
```

Comment:
this is more of a suggestion, editing the same button twice can be confusing IMO.